test/cfg/test_cli.py

```diff
@@ -4,5 +4,16 @@
 
 def test_info():
     cmd = 'python -m recs --info'.split()
-    r = sp.run(cmd, text=True, check=True, stdout=sp.PIPE).stdout
-    json.loads(r)
+    try:
+        r = sp.run(cmd, text=True, check=True, stdout=sp.PIPE).stdout
+    except sp.CalledProcessError:
+        # On Windows, Python installation can also be via the Windows store
+        # When installed in this way, the python.exe acts as a "launcher" for the Python interpreter
+        # It may not behave like a standalone executable, which can lead to issues
+        # Especially when invoking subprocess commands.
+        # A safety check is to ask the subprocess to run in a shell
+        # This is an "except" case because not all Python installations are via Windows store
+        # And it'll be more overhead unless absolutely necessary
+        r = sp.run(cmd, text=True, check=True, stdout=sp.PIPE, shell=True).stdout
+    finally:
```

Comment:
Quibble: you don't really need the `finally` clause here, though it isn't wrong.

If line 17 fails, then the test fails, and there's no need to continue with it in the `finally` branch.